packages/venia-concept/src/actions/catalog/asyncActions.js

```diff
@@ -18,7 +18,6 @@ export const getAllCategories = () =>
 export const setCurrentPage = payload =>
     async function thunk(dispatch) {
         dispatch(actions.setCurrentPage.receive(payload));
-        window.scrollTo(0, 0);
```

Comment:
Scrolling the window is a concern of the UI, not the action handler. See the corresponding change in `category.js`.

Reply:
Having `window.scrollTo()` in this particular action was a bit arbitrary—it only covered this one use case—but I don't actually think there's anything wrong with it being part of an async action. Ultimately, async actions are a natural place for side effects to live, and scrolling the window is just a side effect.